model/riscv_insts_zicbom.sail

```diff
@@ -0,0 +1,81 @@
+/* ****************************************************************** */
+/* This file specifies the instructions in the Zicbom extension */
+
+
+/* ****************************************************************** */
+union clause ast = RISCV_ZICBOM : (cbop_zicbom, regidx)
+
+mapping encdec_cbop : cbop_zicbom <-> bits(12) = {
+  CBO_CLEAN <-> 0b000000000001,
+  CBO_FLUSH <-> 0b000000000010,
+  CBO_INVAL <-> 0b000000000000
+}
+
+mapping clause encdec = RISCV_ZICBOM(cbop, rs1) if haveZicbom()
+  <-> encdec_cbop(cbop) @ rs1 @ 0b010 @ 0b00000 @ 0b0001111 if haveZicbom()
+
+mapping cbop_mnemonic : cbop_zicbom <-> string = {
+  CBO_CLEAN <-> "cbo.clean",
+  CBO_FLUSH <-> "cbo.flush",
+  CBO_INVAL <-> "cbo.inval"
+}
+
+mapping clause assembly = RISCV_ZICBOM(cbop, rs1)
+  <-> cbop_mnemonic(cbop) ^ spc() ^ reg_name(rs1)
+
+val process_clean_inval : (xlenbits, xlenbits, bool, bool) -> Retired effect {escape, rmem, rmemt, rreg, wmv, wmvt, wreg}
+function process_clean_inval(vaddr, width, clean, inval) = {
+  match ext_data_get_addr(zeros(), vaddr, Read(Data), BYTE) {
```

Comment:
As I'm pretty sure I said before, ext_data_get_addr must take the base register that yielded that address in the first argument, not zeroes. That is, you need to pass rs1 (the index) through to here.